File: solutions/3493-maximum-number-of-operations-to-move-ones-to-the-end/maximum-number-of-operations-to-move-ones-to-the-end.java
public class Solution {
    public int maxOperations(String s) {
        int x = 0, c = 0;
        int n = s.length();
        for (int i = 0; i < n; i++) {
            if (s.charAt(i) == '1') {
                c++;
            } else if (s.charAt(i) == '0') {
                if (i == n - 1 || s.charAt(i + 1) == '1') {
                    x += c;
                }
            }
        }
        return x;
    }
}

File: solutions/3493-maximum-number-of-operations-to-move-ones-to-the-end/maximum-number-of-operations-to-move-ones-to-the-end.py
class Solution:
    def maxOperations(self, s: str) -> int:
        x = 0
        c = 0
        n = len(s)
        for i in range(n):
            if s[i] == '1':
                c += 1
            elif s[i] == '0':
                if i == n - 1 or s[i + 1] == '1':
                    x += c
        return x